data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,588 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	"Okay," says Anna, "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and meet me there."`
+
+
+mission "Coalition: Struggling Author 2"
+	name "Help Anna Register"
+	description "Take Anna Fangvin to the <destination> so she can register with the Heliarchs."
+	source "Ki Patek Ka"
+	destination "Ring of Friendship"
+	passengers 1
+	to offer
+		has "Coalition: Struggling Author 1: done"
+	on offer
+		conversation
+			`You walk through the spaceport, looking for Anna. You find her in the middle of a public square, seated at a table with her laptop out. You figure she must've intended on getting started on some early work, what with the newly found inspiration of being in such an alien society, but it seems the sheer volume and diversity of peoples here on <origin> made her forget to type entirely.`
+			`	She only seems to notice you when you also sit down beside her. "Oh, hello again, <first>!" she says, snapping back into focus. "Is the ship ready?"`
+			`	Hearing an affirmative answer, Anna stands up from her chair and begins following you to the ship. She settles into her bunk as you prepare for a trip to the <destination>.`
+				accept
+	on enter "Quaru"
+		conversation
+			`As you enter <system>, Anna walks out of her bunk and finds a spot on the side of the bridge, peering into the empty void. "What a massive ringworld!" she exclaims.`
+			choice
+				`	"It's enormous. As you know, they stole it from the Quarg roughly seven thousand years ago."`
+				`	(Say nothing.)`
+			`	"This is going to be a great source of inspiration for my next novel," says Anna, still in awe. She pulls out a small notebook and begins busily writing in it.`
+	on complete
+		conversation
+			`Anna is amazed as you touch down on the ringworld. As the two of you open the hatch, you see a Heliarch agent in front of you.`
+			`	"Greetings, humans. For you, what can I do today?" asks the Heliarch. You respond that you would like to register Anna with the Heliarchy. After reaching the registration office, Anna enters it, coming out after fifteen minutes. "They finally got all the paperwork done. I'm off to the spaceport to get some food. Meet me there to discuss our next move."`
+
+
+mission "Coalition: Struggling Author 3"
+	name "Back to <planet>"
+	description "Now that Anna Fangvin has completed the registration process, help her get a translation device on <destination>."
+	source "Ring of Friendship"
+	destination "Ki Patek Ka"
+	passengers 1
+	to offer
+		has "Coalition: Struggling Author 2: done"
+	on offer
+		conversation
+			`Going through the spaceport looking for Anna, you finally find her at a restaurant serving food made for Saryds. She's typing away furiously at her laptop, with a plate of vegetables next to her. She sees you and motions for you to take a seat.`
```

Comment:
It could read as if Anna was serving the food otherwise.
```suggestion
			`Going through the spaceport looking for Anna, you finally find her at a restaurant, eating food made for Saryds. She's typing away furiously at her laptop with a plate of vegetables next to her. She sees you and motions for you to take a seat.`
```